testproxy/services/create-client.js

```diff
@@ -54,12 +61,33 @@ const createClient = ({clientMap}) =>
     if (callCredential && callCredential.jsonServiceAccount) {
       authClient = JSON.parse(request.callCredential.jsonServiceAccount);
     }
+    if (request.perOperationTimeout) {
+      /**
+       * This block of code ensures the server times out for every method call
+       * after the amount of time specified in request.perOperationTimeout.
+       */
+      Object.entries(
+        clientConfig.interfaces['google.bigtable.v2.Bigtable'].methods
+      ).forEach(([k, v]) => {
+        if (k === 'ReadRows') {
+          /*
+          TODO: In the future we should apply this for all methods, but right
```

Comment:
If a tracking bug isn't open for this in buganizer please open one!

Reply:
https://b.corp.google.com/issues/383094924